pkg/controllers/vdb/version_reconciler_test.go

```diff
@@ -33,6 +33,9 @@ var _ = Describe("k8s/version_reconcile", func() {
 
 	It("should update annotations in vdb since they differ", func() {
 		vdb := vapi.MakeVDB()
+		vdb.ObjectMeta.Annotations = map[string]string{
```

Comment:
Why did you add this?

Reply:
For this test, 'vertica-k8s:latest' is deployed using the Vcluster style, not the AT style. If we use the default False annotation with Vcluster deployment, this test will fail. I have set the annotation to true so they can pass verification for the image. Is there any way to deploy 'vertica-k8s:latest' using the AT style?